go/cmd/vtbackup/cli/vtbackup.go

```diff
@@ -335,6 +340,14 @@ func takeBackup(ctx, backgroundCtx context.Context, topoServer *topo.Server, bac
 	if err != nil {
 		return fmt.Errorf("failed to initialize mysql config: %v", err)
 	}
+	ctx, cancelCtx := context.WithCancel(ctx)
+	backgroundCtx, cancelbackgroundCtx := context.WithCancel(backgroundCtx)
+	mysqld.OnFailure(func(err error) {
+		log.Warning("Cancelling the vtbackup context as MySQL has failed")
```

Comment:
```suggestion
		log.Warningf("Cancelling the vtbackup context as MySQL has failed: %v", err)
```

Reply:
We are now (93d6c5273e0972f3ff56a942a30cadddca29fb6c) using `OnTerm`, there are no error passed through that we can log.